src/common/scss/form-input.scss

```diff
@@ -127,3 +127,10 @@ textarea {
     margin-right: 8px;
   }
 }
+
+.description-text {
+  @include typography.type-ui-02;
+  color: var(--kd-color-text-primary);
+  margin-top: -8px;
```

Comment:
From the design, it looks like this should be -4px (4px spacing between label and description).

Reply:
Addressed.